evaluation/tests.py

```diff
@@ -0,0 +1,148 @@
+"""
+Project OCELoT: Open, Competitive Evaluation Leaderboard of Translations
+"""
+import os
+from datetime import datetime
+
+from django.test import TestCase
+from django.utils import timezone
+
+from leaderboard.models import Competition
+from leaderboard.models import Language
+from leaderboard.models import SGML_FILE
+from leaderboard.models import Submission
+from leaderboard.models import Team
+from leaderboard.models import TestSet
+from leaderboard.models import TEXT_FILE
+from ocelot.settings import BASE_DIR
+
+TESTDATA_DIR = os.path.join(BASE_DIR, 'leaderboard/testdata')
```

Comment:
@snukky: start using `pathlib.Path` instead? We are on Python 3 anyways, so this should be available. Makes paths much more readable...

Reply:
I agree that `pathlib` will be a better choice. I will open a new PR for this.